packages/contracts/contracts/bridge/hyperlane/interfaces/IInterchainSecurityModule.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
This is just preference but probably easier to maintain by importing @hyperlane-xyz/core than vendor the contracts?